LICENSE_FILE_HEADER

```diff
@@ -0,0 +1,15 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
```

Comment:
Are we required to keep these lines in every file or can we get away with the SPDX ID?

Reply:
Yeah I think we need those.